src/api/post-user.jsx

```diff
@@ -0,0 +1,30 @@
+import { collection, addDoc } from 'firebase/firestore/lite';
+import { db } from '../firebase/firebase_config';
+
+export default async function postCustomer(
+  schoolEmail,
+  password,
+  school,
+  preferredContact,
+  secondaryEmail,
+  phone,
+  paymentMethod,
+  venmo
+) {
+  // @todo Error check for duplicate schoolEmail
+  try {
+    const newCustomer = await addDoc(collection(db, 'customers'), {
+      schoolEmail,
+      password,
+      school,
+      preferredContact,
+      secondaryEmail,
+      phone,
+      paymentMethod,
+      venmo
+    });
+    return newCustomer;
+  } catch (e) {
+    throw new Error('Issue in creating account');
```

Comment:
Should we send the error that was caught so the user has an understanding of what the error is instead of a generic Error message?

Reply:
In my new commit I hopefully deal with this? I still have the generic issue in creating account error for the API call specifically but there other errors (such as duplicate emails)

Reply:
Yea that looks good. I was thinking of also throwing the error caught in the API call but on second thought, maybe we can just log that.